docs/fsharp/language-reference/byrefs.md

```diff
@@ -176,14 +176,19 @@ Although these rules very strongly restrict usage, they do so to fulfill the pro
 Byref returns from F# functions or members can be produced and consumed. When consuming a `byref`-returning method, the value is implicitly dereferenced. For example:
 
 ```fsharp
-let safeSum(bytes: Span<byte>) =
-    let mutable sum = 0
+let squareAndPrint (data : byref<int>) = 
+    let squared = data*data    // data is implicitly implicitly dereferenced
+    printfn "%d" squared
+```
+To return a value byref, the variable which contains the value must live longer than the current scope.
```

Comment:
```suggestion
To return a byref of a value, the value needs to be declared in an outer scope than the one it is used in.
```

Reply:
I've added a blank line to your suggestion @cartermp to fix the linting error at the same time.